tests/sqllogictests/suites/query/window_function/window_in_expr.test

```diff
@@ -78,6 +78,24 @@ from
 ----
 1
 
+statement ok
+CREATE OR REPLACE TABLE sales (
+    sale_id INT UNSIGNED NOT NULL,
+    product_id INT UNSIGNED NOT NULL,
+    customer_id INT UNSIGNED NOT NULL,
+    sale_date DATE NOT NULL,
+    quantity INT NOT NULL,
+    net_paid DECIMAL(10, 2) NOT NULL
+)
+
+statement ok
+SELECT  net_paid,
+      truncate( SUM(net_paid) OVER (PARTITION BY sale_id ORDER BY sale_date), 4)   AS running_avg
```

Comment:
`SUM` --> `AVG`?

Reply:
it's not related to sum or avg.

Reply:
Yes, it like.

From my test:
1. works
```
SELECT customer_id, sale_id, net_paid,
       SUM(net_paid) OVER (PARTITION BY customer_id ORDER BY sale_date) AS running_total,
        truncate(AVG(net_paid) OVER (PARTITION BY customer_id ORDER BY sale_date), 4) AS running_avg
        ... ...
```

2. works
```
SELECT customer_id, sale_id, net_paid,
       truncate(SUM(net_paid) OVER (PARTITION BY customer_id ORDER BY sale_date), 4)  AS running_total,
       AVG(net_paid) OVER (PARTITION BY customer_id ORDER BY sale_date) AS running_avg
        ... ...
```

Not working:
```
SELECT customer_id, sale_id, net_paid,
       truncate(SUM(net_paid) OVER (PARTITION BY customer_id ORDER BY sale_date), 4)  AS running_total,
       truncate(AVG(net_paid) OVER (PARTITION BY customer_id ORDER BY sale_date), 4) AS running_avg
        ... ...
```